File: mario-more-sentimental/mario.py
# TODO

# Import libraries
from cs50 import get_int

# Ask user for desired height
while True:
    h = get_int("Height: ")
    if (h > 0 and h <= 8):
        break

# Iterate through rows and columns
for i in range(1, 1 + h):
    for j in range(1, 1 + h):

        # Print left side
        if j > (h - i):
            print("#", end="")
        else:
            print(" ", end="")

        # Print right side
        if j == h:
            print(" ", "#" * i, end="")

    print()

File: mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int n;

    printf("\n");

    // Ask for size
    do
    {
        n = get_int("Size: ");
    }
    while (n < 1 || n > 8);

    // Each Row
    for (int i = 0; i < n; i++)
    {
        // Printing Columns
        for (int j = 0; j < n * 2 + 1; j++)
        {
            if (j < n - i - 1)
            {
                printf(" ");
            }
            else if (j == n)
            {
            printf("  ");
            }
            else if (j < n + i + 2)
            printf("#");
        }
        // Next Row
        printf("\n");
    }
}